C/tests/c4DatabaseTest.cc

```diff
@@ -965,8 +965,21 @@ TEST_CASE("Database Upgrade From 2.7", "[Database][Upgrade][C]") {
 }
 
 
+TEST_CASE("Database Upgrade From 2.7 to Version Vectors", "[Database][Upgrade][C]") {
+    testOpeningOlderDBFixture("upgrade_2.7.cblite2", kC4DB_VersionVectors);
+}
+
+
+TEST_CASE("Database Upgrade From 2.8", "[Database][Upgrade][C]") {
```

Comment:
This test is moved here from feature/deleted-keystore. I changed master branch's upgrade_2.8.cblite2 to upgrade_2.8_index.cblite2